helpers/lib/cdnify.js

```diff
@@ -56,7 +56,14 @@ module.exports = globals => {
             }
 
             if (protocol === 'webdav:') {
-                return [cdnUrl, 'content', path].join('/');
+                const supportedImageExtensions = ['jpg', 'jpeg', 'gif', 'png'];
+                const unsupportedPathSyntaxes = ['../', './'];
+                const searchParamIgnoredPath = path.split('?')[0];
+                const isImage = supportedImageExtensions.some(ext => searchParamIgnoredPath.toLowerCase().endsWith(ext));
```

Comment:
previously used method `includes()` can lead to incorrect results. For example, if  provided path looks like `'webdav:banners/menu-giftcard.svg'` then we will get situation where  `path.toLowerCase().includes(ext))` return `true` and helper modified path for unsupported content.